Packs/PAN-OS/Integrations/Panorama/Panorama_test.py

```diff
@@ -6215,6 +6215,26 @@ def test_get_parsed_incident_entries(
         mocker.patch('Panorama.parse_incident_entries', return_value=fetch_incidents_input.one_incident_result)
         assert get_parsed_incident_entries(incident_entries_dict, last_fetch_dict, last_id_dict) == expected_result
 
+    @patch("Panorama.GET_LOG_JOB_ID_MAX_RETRIES", 1)
+    @pytest.mark.parametrize('response, debug_msg, expected_result',
+                             fetch_incidents_input.get_query_entries_by_id_request_args)
+    def test_get_query_entries_by_id_request(self, mocker, response, debug_msg, expected_result):
+        """
+        Given:
+        - a valid query id
+
+        When:
+        - get_query_entries_by_id_request function is called
+
+        Then:
+        - assert that the returned response is valid, or retry if the job is not finished
```

Comment:
Please explain here in a few words about each tested case, 
Like: 
"""
        Given:
        1.  A valid query id with a success (FIN status)
        2. A valid query id... 
        3. etc
        When:
        - get_query_entries_by_id_request function is called
        Then:
        assert that the returned response is valid, or retry if the job is not finished
        Verify that: 
        1. ...
        2. ...
        3. ...
        4. ...

Reply:
Done.